theme.json

```diff
@@ -224,49 +231,49 @@
 					"fontFace": [
 						{
 							"src": [
-								"file:./assets/fonts/literata/Literata-BlackItalic.woff2"
+								"file:./assets/fonts/literata/Literata60pt-ExtraLightItalic.woff2"
 							],
-							"fontWeight": "900",
+							"fontWeight": "200",
 							"fontStyle": "italic",
 							"fontFamily": "Literata"
 						},
 						{
 							"src": [
-								"file:./assets/fonts/literata/Literata-Bold.woff2"
+								"file:./assets/fonts/literata/Literata-LightItalic.woff2"
 							],
-							"fontWeight": "700",
-							"fontStyle": "normal",
+							"fontWeight": "300",
+							"fontStyle": "italic",
 							"fontFamily": "Literata"
 						},
 						{
 							"src": [
-								"file:./assets/fonts/literata/Literata-BoldItalic.woff2"
+								"file:./assets/fonts/literata/Literata60pt-Light.woff2"
 							],
-							"fontWeight": "700",
-							"fontStyle": "italic",
+							"fontWeight": "300",
+							"fontStyle": "normal",
 							"fontFamily": "Literata"
 						},
 						{
 							"src": [
-								"file:./assets/fonts/literata/Literata-ExtraBold.woff2"
+								"file:./assets/fonts/literata/Literata60pt-LightItalic.woff2"
```

Comment:
It looks like this one has the same configuration as [this one](https://github.com/WordPress/twentytwentyfive/pull/101/files#diff-160c735a986cf8c05e1f6b2e9ac89dd0410562d2fe5ccbdc93157bd92f3dc2bcR242).

Reply:
It might be because of the name are different. I'll check them manually again and fix the duplicates.